packages/typespec-test/test/todo_non_branded/generated/typespec-ts/src/api/todoItems/index.ts

```diff
@@ -401,6 +401,6 @@ export function list(
     () => _listSend(context, options),
     _listDeserialize,
     ["200"],
-    { itemName: "items" },
+    { itemName: "items", nextLinkName: "nextLink" },
```

Comment:
This will be not an issue for todo cases because the value is the same as our default next link.